src/entity/in_memory.ts

```diff
@@ -52,15 +52,16 @@ export class InMemoryEntity {
             : clone(config);
     }
 
-    prop<T = null>(name: string, defaultValue?: T): T;
+    prop<T = undefined>(name: string, defaultValue: T): T;
+
+    prop<T = undefined>(name: string): T | undefined;
 
     /**
      * @summary Return a prop or the default
-     * @returns {*}
      */
-    prop<T>(name: string, defaultValue: T | null = null): unknown {
+    prop<T = undefined>(name: string, defaultValue?: T): T | undefined {
```

Comment:
Here I replaced default return value from `null` to `undefined`
The `prop()` must return `undefined` instead of `null`.
Next problem happened in web-app's class ServiceLevel:
```ts
get expirationPeriodType() {
     return this.prop("expirationPeriodType");
}

setExpirationTypeAndDate(type = "monthly") {
   // expect the `type` prop to be "monthly" if empty, but receive `null`
}

serviceLevel.setExpirationTypeAndDate(serviceLevel.expirationPeriodType);
```